Mathlib/RingTheory/Polynomial/SmallDegreeVieta.lean

```diff
@@ -56,20 +61,26 @@ lemma eq_mul_mul_of_aroots_quadratic_eq_pair [CommRing T] [CommRing S] [IsDomain
     X ^ 2 + C ((algebraMap T S) b) * X + C ((algebraMap T S) c) by simp] at haroots
   exact eq_mul_mul_of_roots_quadratic_eq_pair haroots
 
+lemma quadratic_eq_of_vieta [CommRing R] {a b c x1 x2 : R}
+    (hvieta : b = -a * (x1 + x2) ∧ c = a * x1 * x2) :
```

Comment:
We do not package assumptions together using conjunction like this as it is less extensible than making them separate assumptions:
```suggestion
    (hsum : b = -a * (x1 + x2)) (hprod : c = a * x1 * x2) :
```
(similarly below)